framework/src/media/audio/audio_manager.c

```diff
@@ -484,27 +487,66 @@ static int get_actual_audio_out_card_id()
 	return 0;
 }
 
+static int resample_stream_in(struct pcm *pcm, audio_card_info_t *cur_card, void *data, uint32_t input_frames)
```

Comment:
before call resample_stream_in & out we check pcm's validation.
So pcm is not neccessary here.

Reply:
You're right. Removed.